modules/qtwidgets/src/angleradiuswidget.cpp

```diff
@@ -30,8 +30,9 @@
 #include <modules/qtwidgets/angleradiuswidget.h>
 
 #include <algorithm>  // for max, min
-#include <cmath>      // for atan2, cos, sin, sqrt, M_PI
+#include <cmath>      // for atan2, cos, sin, sqrt, std::numbers::pi
 #include <sstream>    // for stringstream, basic_ostream, fixed
+#include <numbers>
```

Comment:
```suggestion
#include <cmath>      // for atan2, cos, sin, sqrt
#include <sstream>    // for stringstream, basic_ostream, fixed
#include <numbers>    // std::numbers::pi
```

Reply:
Will rerun the include-what-you-use periodically to fix those things...